wrapping/itkAnalyticSignalImageFilter.wrap

```diff
@@ -0,0 +1,10 @@
+itk_wrap_class("itk::AnalyticSignalImageFilter" POINTER)
+  foreach(d ${ITK_WRAP_IMAGE_DIMS})
+    foreach(rt ${WRAP_ITK_REAL})
+      foreach(ct ${WRAP_ITK_COMPLEX_REAL})
```

Comment:
>../../../include/itkVnlForward1DFFTImageFilter.hxx:100:24: error: no viable conversion from 'complex<double>' to 'const complex<float>'

Try wrapping only `float` with `complex<float>` and `double` with `complex<double>`

Reply:
Ah that makes sense. Looks like it fixed warnings on my end, waiting to see what CI says.